backend/cmd/migrate/migrations/000005_create_topology_cache_table.up.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE topology_cache (
```

Comment:
existing migration files should never be modified. they preserve that version of the table. they are also run on any new empty table and modifying it would render the later alter migration incorrect.

only use subsequent migrations to alter existing definitions.

Reply:
i know it was left from my testing.